src/native/tab_bar.rs

```diff
@@ -4,18 +4,28 @@
 //! to use the [`Tabs`](super::tabs::Tabs) widget instead.
 //!
 //! *This API requires the following crate features to be activated: `tab_bar`*
-use iced_native::{touch, Element};
-use std::hash::Hash;
-
 use iced_native::{
-    column, event, layout, mouse, row, text, Alignment, Clipboard, Column, Event, Font, Hasher,
-    Layout, Length, Point, Rectangle, Row, Text, Widget,
+    alignment::{Horizontal, Vertical},
+    event,
+    layout::{Limits, Node},
+    mouse, renderer, touch,
+    widget::{Column, Row, Text},
+    Alignment, Clipboard, Color, Element, Event, Font, Layout, Length, Point, Rectangle, Widget,
 };
 
 pub mod tab_label;
 pub use tab_label::TabLabel;
 
-use crate::core::renderer::DrawEnvironment;
+pub use crate::{
+    graphics::icons,
```

Comment:
I don't think that republishing the `graphics::icons` is necessary here, as the icons are published to the end-user somewhere else. A normal `use ...` should work as well.

Reply:
oh true ill change that.